src/build.h

```diff
@@ -219,7 +219,7 @@ struct Builder {
 #else
   unique_ptr<CommandRunner> command_runner_;  // auto_ptr was removed in C++17.
 #endif
-  BuildStatus* status_;
+  Status* status_;
```

Comment:
I would prefer to see the lifetime of this managed properly.

Is this the only place that has a pointer to the Status object?

If so, can we wrap it in auto_ptr / unique_ptr, just like the CommandRunner above?